### ISSUE TITLE
[Unholy Death Knight] Verify pet is active

### ISSUE THREAD

**Issue by jjs451** [WoWAnalyzer/WoWAnalyzer#3404]
I received a feature request to add a check to ensure pet is active and alive during the fight.  Adding it as an issue here as a reminder.

**Comment by torothin** [comment]
By "active" do you mean the pet is attacking when attacking is available?

**Comment by jjs451** [comment]
Yes.  The request I got was to make sure pet was doing damage during the encounter

**Comment by darthelit** [comment]
Is anyone working on this? I finally fixed my machine and can work on this again.

**Comment by jjs451** [comment]
@darthelit I don't think anyone is
